Can't build Windows images

**Description**

The build process fails because of a missing declaration of common variables. The variables were removed in #300 (commit 63fc44e).


**To Reproduce**

- Try to build any Windows image (for example: `make windows10Pro`)
- See the failed build in the output:
```
user@950372497cbf:/one-apps-builder/one-apps$ make windows10Pro
packer/build.sh 'windows' '10Pro' export/windows10Pro.qcow2

No plugins requirement found, make sure you reference a Packer config
containing a packer.required_plugins block. See
https://www.packer.io/docs/templates/hcl_templates/blocks/packer
for more info.
Error: Undefined -var variable

A "input_dir" variable was passed in the command line but was not found in known
variables. To declare variable "input_dir", place this block in one of your .pkr
files, such as variables.pkr.hcl

Error: Undefined -var variable

A "output_dir" variable was passed in the command line but was not found in
known variables. To declare variable "output_dir", place this block in one of
your .pkr files, such as variables.pkr.hcl

Error: Undefined -var variable

A "headless" variable was passed in the command line but was not found in known
variables. To declare variable "headless", place this block in one of your .pkr
files, such as variables.pkr.hcl

Error: Undefined -var variable

A "arch" variable was passed in the command line but was not found in known
variables. To declare variable "arch", place this block in one of your .pkr
files, such as variables.pkr.hcl

Error: Undefined -var variable

A "appliance_name" variable was passed in the command line but was not found in
known variables. To declare variable "appliance_name", place this block in one
of your .pkr files, such as variables.pkr.hcl

Error: Undefined -var variable

A "distro" variable was passed in the command line but was not found in known
variables. To declare variable "distro", place this block in one of your .pkr
files, such as variables.pkr.hcl

Error: Undefined -var variable

A "version" variable was passed in the command line but was not found in known
variables. To declare variable "version", place this block in one of your .pkr
files, such as variables.pkr.hcl


make: *** [Makefile:74: export/windows10Pro.qcow2] Error 1
user@950372497cbf:/one-apps-builder/one-apps$ 
```

**Expected behavior**

The Windows build process should start

**Details**

 - Affected Component: Windows packer images
 - Hypervisor: KVM
 - Version: master branch, commit 63fc44e

**Additional context**

The root cause of this issue was #286 and commit c87b10b05b9d36cf9cafc5bb970d7d0a881a847a, which removed the *packer/windows/common.pkr.hcl* symlink to *packer/common.pkr.hcl* file and did not replace it with a symlink to new *packer/vars-common.pkr.hcl*. However, the build process continued to work because of redundant variable definitions in *packer/windows/variables.pkr.hcl*, which were finally removed in #300.

## Progress Status

- [ ] Code committed
- [ ] Testing - QA
- [ ] Documentation (Release notes - resolved issues, compatibility, known issues)

--- a/packer/windows/vars-common.pkr.hcl
+++ b/packer/windows/vars-common.pkr.hcl
@@ -1,0 +1,1 @@
+../vars-common.pkr.hcl

